src/eos-autoupdater.c

```diff
@@ -59,7 +59,7 @@ typedef enum _UpdateStep {
 #define SEC_PER_DAY (3600ll * 24)
 
 /* This file is touched whenever the updater starts */
-static const char *UPDATE_STAMP_DIR = LOCALSTATEDIR "/eos-updater";
+static const char *UPDATE_STAMP_DIR = SHAREDSTATEDIR "/eos-updater";
```

Comment:
Apparently, I didn’t built with my latest changes. That’s quite embarrassing. Let’s go with the `localstatedir` precedent then. Fixed as df5e7ba (I’ll rewrite the commit message when squashing it into the other commit).